release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -41,19 +41,36 @@ This release adds improvements related to the following components and concepts.
 === {op-system-first}
 
 [id="ocp-4-7-enhanced-disk-provisioning"]
-==== Enhanced disk provisioning for LUKS and RAID devices
+==== Enhanced disk provisioning for LUKS, RAID, and FBA DASD
 
-{op-system} 4.7 includes several improvements to disk provisioning for bare metal deployments:
+{product-title} 4.7 includes several improvements to disk provisioning for bare metal deployments:
 
 * Native Ignition support for LUKS disk encryption provides additional configurability for encrypted root filesystems, as well as support for encryption of additional data filesystems.
 * {op-system} now supports boot disk mirroring, except on s390x, providing redundancy in the case of disk failure.
 * {op-system} on s390x can be installed onto FBA DASD disks.
 
 [NOTE]
 ====
-LUKS configuration can no longer be performed in the `/etc/clevis.json` file. Provisioning fails if `/etc/clevis.json` is specified in the machine config.
+On new clusters, LUKS configuration must use the native Ignition mechanism, as provisioning fails if the legacy `/etc/clevis.json` file is included in the machine config. On clusters that are upgrading from {product-title} 4.6 or earlier, LUKS can only be configured by using `/etc/clevis.json`.
 ====
 
+* Multipath encryption is now available for root devices attached to clusters that are created using {product-title} 4.7 or higher. This is a useful day-2 feature that can be set up by configuring the machine config, such as shown in the following example:
```

Comment:
Multipath isn't related to encryption (though both can be used together).

Reply:
Thanks, @bgilbert - I reworded this part based on your comments. I could not find a "storage provisioning" section that makes sense for adding the `MachineConfig` example with the multipath karg.

Leaving this comment here for you and @Prashanth684. If you have a specific place in the docs where you think it might make sense, let me know and I can open a separate PR to include the example. 

We do have a [Post-installation storage configuration](https://docs.openshift.com/container-platform/4.7/post_installation_configuration/storage-configuration.html) section, but that's really just a single-sourced collection from our core [Storage](https://docs.openshift.com/container-platform/4.7/storage/understanding-persistent-storage.html) docs.

Reply:
@bobfuru so sorry i missed this completely! I think it needs to be mentioned that multipath by default is applicable to all devices, not just the root device. The example with the machine config shows how to multipath the root device.

Reply:
Thank you, @Prashanth684 - I have made the change in https://github.com/openshift/openshift-docs/pull/28894.